hoodie-common/src/main/java/com/uber/hoodie/common/table/timeline/HoodieActiveTimeline.java

```diff
@@ -97,7 +97,7 @@ protected HoodieActiveTimeline(FileSystem fs, String metaPath, String[] included
     public HoodieActiveTimeline(FileSystem fs, String metaPath) {
         this(fs, metaPath,
             new String[] {COMMIT_EXTENSION, INFLIGHT_COMMIT_EXTENSION, DELTA_COMMIT_EXTENSION,
-                INFLIGHT_DELTA_COMMIT_EXTENSION, COMPACTION_EXTENSION,
+                INFLIGHT_DELTA_COMMIT_EXTENSION, COMPACTION_EXTENSION, INFLIGHT_ROLLBACK_EXTENSION, ROLLBACK_EXTENSION,
```

Comment:
Yeah, I looked at all methods in HoodieActiveTimeline, they all filter the instants by an action or a set of actions passed in so adding instants should not affect any behavior of HoodieActiveTimeline. All methods in HoodieDefaultTimeline also stream back all instants (as requested) so it's behavior should not be affected. The size of instants will grow but given we are adding the rollback instant, it should not blow up the size considerably. 
I don't see other ways instants are being accessed. Anything else we should look into ?